### PR TITLE
Fix form UI

### PR DESCRIPTION
**Pull request by IlliaDerevianko**
- Show active links in desktop navbar
- Show active links in mobile navbar
- Removed an extra option for opportunity type when creating/editing
- Fix Skills/Interests chips so that they don't move out of the screen

Screenshot 2024-08-28 at 12 42 55

Screenshot 2024-08-28 at 12 42 30

### CODE REVIEW

File: components/navbar/DesktopNavbar.tsx
Comment (alexanderbira) [inline components/navbar/DesktopNavbar.tsx:1]: Make the login link stay underlined if the user is on the login page

**Review by alexanderbira — CHANGES_REQUESTED** [review]
Great functionality!

File: components/navbar/DesktopNavbar.tsx
Comment (matalex412) [reply components/navbar/DesktopNavbar.tsx:1]: Resolved in [b8976b6](https://github.com/imperial/cpp-connect/pull/79/commits/b8976b6ceb28217c91497b4e75b489c83f87f8af)

File: components/navbar/DesktopNavbar.tsx [hunk 1]
```diff
@@ -57,9 +58,10 @@ const DesktopNavbar = (props: NavbarProps) => {
           {isSignedIn(data, props) ? (
             <ProfileDropdown {...props} />
           ) : (
-            <Link href="/auth/login" className={styles.link}>
-              <span>Log In</span>
-            </Link>
+            // <Link href="/auth/login" className={styles.link}>
+            //   <span>Log In</span>
+            // </Link>
```
Comment (alexanderbira) [inline components/navbar/DesktopNavbar.tsx:63]: Remove comment
```suggestion
```

**Review by alexanderbira — CHANGES_REQUESTED** [review]

**Review by alexanderbira — APPROVED** [review]

**Pull request merged**